### DISCUSSION TITLE
Issue with new docker image releases: IBM Db2 Error for libdb2.so

### DISCUSSION THREAD

**Discussion by SaraJonnas** [dotnet/dotnet-docker#4566] [General]
Hi @lbussell,

With every release of dockerimage for asp.net we get libdb2.so not found error. We tried several different ways but no luck. all our projects have dependencies on IBM DB2 drivers. We were using 7.0.3 bullseye and now its not supported. Please advice.

Thanks
Sarada J

**Comment by mthalman** [comment]
What does your Dockerfile look like? The .NET images don't contain `libdb2.so` by default so it would help to have some context in what you're doing in your Dockerfile.
